packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3265,38 +3273,54 @@ export default class Meeting extends StatelessWebexPlugin {
 
           if (this.config.stats.enableStatsAnalyzer) {
             // TODO: ** Dont re create StatsAnalyzer on reconnect or rejoin
-            this.statsAnalyzer = new StatsAnalyzer(this.config.stats);
+            this.networkQualityMonitor = new NetworkQualityMonitor(this.config.stats);
+            this.statsAnalyzer = new StatsAnalyzer(this.config.stats, this.networkQualityMonitor);
             this.statsAnalyzer.on(EVENT_TRIGGERS.MEDIA_QUALITY, (options) => {
               // TODO:  might have to send the same event to the developer
               // Add ip address info if geo hint is present
               options.data.intervalMetadata.peerReflexiveIP = this.webex.meetings.geoHintInfo?.clientAddress || options.data.intervalMetadata.peerReflexiveIP || MQA_STATS.DEFAULT_IP;
               Metrics.postEvent({event: eventType.MEDIA_QUALITY, meeting: this, data: {intervalData: options.data, networkType: options.networkType}});
             });
-            this.statsAnalyzer.on(EVENT_TRIGGERS.MEETING_HIGH_PACKETLOSS, (res) => {
-              if (res.mediaType === VIDEO) {
-                if (this.config.stats.autoDowngradeEnabled && this.canUpdateMedia()) {
-                  LoggerProxy.logger.log(`${LOG_HEADER} High Packet Loss Detected, Adjusting Local Video Quality`);
-                  if (this.mediaProperties.localQualityLevel === QUALITY_LEVELS.HIGH) {
-                    this.setLocalVideoQuality(QUALITY_LEVELS.MEDIUM);
-                  }
-                  else if (this.mediaProperties.localQualityLevel === QUALITY_LEVELS.MEDIUM) {
-                    this.setLocalVideoQuality(QUALITY_LEVELS.LOW);
-                  }
+            this.networkQualityMonitor.on(EVENT_TRIGGERS.NETWORK_QUALITY_JUDGMENT, (res) => {
```

Comment:
should we save the value on the meeting object ?

Reply:
It is currently do you not want to?  for example it would be consumed this way in telehealth
meeting.on('network:quality:judgment', Meetings.handleNetworkQuality.bind(this));

Reply:
or do you mean the event namespace so 
meeting.on('meeting:quality:judgment', Meetings.handleNetworkQuality.bind(this));